runtime/lib/connection/PropelPDO.php

```diff
@@ -647,6 +647,24 @@ public function log($msg, $level = null, $methodName = null, array $debugSnapsho
         }
     }
 
+    /**
+     * Returns a string with files, lines e functions of the stacktrace to this method
+     *
+     * @return str
+     */
+    function generateCallTrace()
+    {
+      $stacktrace = debug_backtrace(2);
+      $stackLog = '';
+      foreach ($stacktrace as $stack) {
+        $file = isset($stack['file']) ? $stack['file'] : "Unknown file";
+        $line = isset($stack['line']) ? $stack['line'] : "Unknown line";
+        $stackLog .= $file.':'.$line.' '.$stack['function']."()\n";
+      }
```

Comment:
May we apply ckls code standards here too?
```suggestion
      $stacktrace = debug_backtrace(2);
      $stackLog   = '';
      foreach ($stacktrace as $stack) {
        $file     = isset($stack['file']) ? $stack['file'] : "Unknown file";
        $line     = isset($stack['line']) ? $stack['line'] : "Unknown line";
        $stackLog .= $file.':'.$line.' '.$stack['function']."()\n";
      }
```